testutil/sample/campaign.go

```diff
@@ -77,10 +77,25 @@ func MainnetAccount(campaignID uint64, address string) campaign.MainnetAccount {
 
 // MsgCreateCampaign returns a sample MsgCreateCampaign
 func MsgCreateCampaign(coordAddr string) campaign.MsgCreateCampaign {
+	coins := Coins()
+	for i := range coins {
+		coins[i].Amount = coins[i].Amount.Add(campaign.DefaultMinTotalSupply)
+	}
```

Comment:
I think we can introduce a `TotalSupply()` method in `sample` that initializes a `Coins` with token in the total supply range

Reply:
I' also adding a `CoinWithRange(min, max)` and `CoinsWithRange(min, max)` to `sample` since I need it for a test and perhaps having sample functions with boundaries for amounts might be useful for other things